daedalus/src/Daedalus/ClientApi.purs

```diff
@@ -27,11 +28,93 @@ import Daedalus.Crypto as Crypto
 
 -- | Resets wallet database (metadata) and wallets/keys
 -- This should be used only in testing
+-- Arguments:
+-- Returns:
+-- Example in nodejs:
```

Comment:
@ksaric it would be great if you could add example how to use this api from nodejs (when I finish with refactor). This way you will also know how to use the api and you will be able to help if something goes bad. It will also serve as a testing and documentation for frontend guys to use HD wallets and spending passwords (As api has changed a lot now)